airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -11,12 +11,12 @@
 
 
 class AddedFieldDefinition(BaseModel):
-    type: Literal['AddedFieldDefinition']
```

Comment:
This was caught by the format of the CDK_BUILD. It seems to be an issue with [this commit](https://github.com/airbytehq/airbyte/commit/5248854f2e94faffb258949f95b44feb84969ba9#diff-bafa388d211eaf50816b949e1d20ff917b9462e154a96a40e6d5ad36b325e26e). I haven't changed anything in this file